web/.gitignore

```diff
@@ -5,6 +5,7 @@ node_modules
 # local env files
 .env.local
 .env.*.local
+.env
```

Comment:
I thought this was already here 🤔  but if not, it should be! lol and `.env` above should not be committed